### PR TITLE
ppsspp: use cmake to build and system ffmpeg

### PR DESCRIPTION
**Pull request by ToKe79**

### CODE REVIEW

**Comment by ToKe79** [comment]
Hi,
this PR (63ab801b3d15c6a56337710b634b975eb6b391b3) did not build correctly for all targets.
Following targets failed (2):
- imx6.cuboxi.arm
- imx6.udoo.arm

Logs and images for testing can be found here: http://nightly.builds.lakka.tv/PRs/pr_732_63ab801/

_This is an automated message_

File: packages/libretro/ppsspp/package.mk [hunk 1]
```diff
@@ -25,44 +25,36 @@ PKG_ARCH="any"
...
 PKG_IS_ADDON="no"
 PKG_AUTORECONF="no"
-PKG_USE_CMAKE="no"
+PKG_USE_CMAKE="yes"
+
+PKG_CMAKE_OPTS_TARGET="-DLIBRETRO=yes \
+                       -DCMAKE_BUILD_TYPE=Release \
+                       -DUSE_FFMPEG=yes \
+                       -DUSE_SYSTEM_FFMPEG=yes \
+                       --target ppsspp_libretro"
```
Comment (natinusala) [inline packages/libretro/ppsspp/package.mk]: `-DARMV7=yes` must be added on arm platforms

**Comment by natinusala** [comment]
Confirmed working on RPi4 (with the -DARMV7 change), as well as system ffmpeg (the videos are still glitchy but it's a libretro issue, at least it doesn't crash now)

**Comment by ToKe79** [comment]
all should be fixed now, including imx6 (separate patch for CMakeLists in the project folder). also resolved conflict with #733

**Comment by ToKe79** [comment]
Hi,
the build of this PR (7a53edd6bae676ab27f63f8e81f1512ae2761924) finished successfully for all 45 targets.
Images for testing and logs are here: http://nightly.builds.lakka.tv/PRs/pr_732_7a53edd/

_This is an automated message_

**Comment by ToKe79** [comment]
Hi,
the build of this PR (30bdb4164c2511c772a5767f6fe4ba25b725cd72) finished successfully for all 45 targets.
Images for testing and logs are here: http://nightly.builds.lakka.tv/PRs/pr_732_30bdb41/

_This is an automated message_

**Pull request merged**